ui/lib/replication/addon/templates/components/mount-filter-config-list.hbs

```diff
@@ -42,7 +42,7 @@
   </div>
 </div>
 {{#each mounts as |mount|}}
-  {{#unless (or mount.local (includes singletonMountTypes mount.type))}}
+  {{#unless (or mount.local (contains mount.type singletonMountTypes))}}
```

Comment:
I thought `contains` only worked in firefox? Does this work?

Reply:
This is `contains` from the `ember-composable-helpers` addon - https://github.com/DockYard/ember-composable-helpers#contains - `includes` here was a custom helper we write, so I've deleted that in favor of `contains` from the addon we were already using.